content/how-to/ci-checks/github-checks/index.md

```diff
@@ -1,11 +1,13 @@
 ---
-title: "Configure GitHub checks"
+title: "Configure GitHub Checks"
```

Comment:
"GitHub Checks" is the proper name for the product at GitHub, otherwise "checks" would be lowercase